### PR TITLE
OSDOCS-5487: Histogram in Traffic Flow view

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.10+
Issue:

https://issues.redhat.com/browse/OSDOCS-5487

Link to docs preview:

[Using the Histogram](https://56931--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/observing-network-traffic.html#network-observability-working-with-trafficflow_nw-observe-network-traffic)
QE review:
- [X] QE has approved this change.

Additional information:

This section is basically to create a deep link from the Release notes. Tooltips thoroughly explain how to use the histogram feature.

### CODE REVIEW

**Comment by skrthomas** [comment]
@jpinsonneau I know we discussed only a release note for the Histogram feature. However, I think it may deserve a mention in this area of the docs too. Wdyt? I defer to you. I will create a release note regardless, but was thinking that having a small mention in the docs would be appropriate. The text is basically in the tooltip text.

**Comment by jpinsonneau** [comment]
> @jpinsonneau I know we discussed only a release note for the Histogram feature. However, I think it may deserve a mention in this area of the docs too. Wdyt? I defer to you. I will create a release note regardless, but was thinking that having a small mention in the docs would be appropriate. The text is basically in the tooltip text.

Why not ! But in that case, I feel we should also document row selection + side panel with details / json tabs
Do you also plan to document Overview & Topology tabs ? :thinking:

**Comment by skrthomas** [comment]
@jpinsonneau Hmmm...I think maybe the scope of the Histogram feature has changed since we last spoke. I scheduled a quick chat on Wednesday for us to synch about the Conversations documentation. Lets also synch about Histogram as well. I was thinking that it was only in Traffic Flow, but I know you were considering adding it to Topology and Overview tabs as well. I guess we wouldn't want to duplicate anything that's consistently the same across the tabs, so maybe having the Histogram section at the same header level as the Network Traffic, Overview, and Topology if it applies to all.

File: modules/network-observability-histogram-trafficflow.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-histogram-trafficflow_{context}"]
+== Using the histogram
+You can click *Show histogram* to display a toolbar view for visualizing the history of flows as a bar chart while remaining within the Loki query limit. The histogram shows the number of logs over time. You can select a part of the histogram to filter the flows data in the table that follows the toolbar.
```
Comment (jpinsonneau) [inline modules/network-observability-histogram-trafficflow.adoc:8]:
```suggestion
You can click *Show histogram* to display a toolbar view for visualizing the history of flows as a bar chart. The histogram shows the number of logs over time. You can select a part of the histogram while remaining within the Loki query limit to filter the flows data in the table that follows the toolbar.
```

The query limit only apply on the table query; not on the histogram one.

File: modules/network-observability-histogram-trafficflow.adoc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-histogram-trafficflow_{context}"]
+== Using the histogram
+You can click *Show histogram* to display a toolbar view for visualizing the history of flows as a bar chart. The histogram shows the number of logs over time. You can select a part of the histogram to filter the flows data in the table that follows the toolbar.
```
Comment (mburke5678) [inline modules/network-observability-histogram-trafficflow.adoc:8]: This doesn't sound right, or maybe it does. I don't know. 

the flow data
the flow's data
the data

Comment (skrthomas) [reply modules/network-observability-histogram-trafficflow.adoc:8]: Hmmmmm I think maybe "...filter the network flow data"

**Pull request merged**